openhands/controller/agent_controller.py

```diff
@@ -1206,14 +1206,13 @@ def _first_user_message(self) -> MessageAction | None:
         Returns:
             MessageAction | None: The first user message, or None if no user message found
         """
-        # Find the first user message from the appropriate starting point
-        user_messages = list(self.event_stream.get_events(start_id=self.state.start_id))
-
-        # Get and return the first user message
         return next(
             (
                 e
-                for e in user_messages
+                for e in self.event_stream.get_events(
+                    start_id=self.state.start_id,
+                    reverse=False,
```

Comment:
```suggestion
```

NIT: reverse is false by default i believe, but i guess doesn't hurt to indicate its the first user message